ee/clickhouse/models/property.py

```diff
@@ -77,17 +167,21 @@ def parse_prop_clauses(
                 "{}person".format(prepend),
                 prop_var="person_props" if is_direct_query else "properties",
                 allow_denormalized_props=allow_denormalized_props and is_direct_query,
+                property_operator=property_operator,
             )
             if is_direct_query:
                 final.append(filter_query)
                 params.update(filter_params)
             else:
+                # Subquery filter here always should be blank as it's the first
+                filter_query = filter_query.replace(property_operator, "", 1)
```

Comment:
@neilkakkar I changed the above "AND" back to the variable so it's less confusing and made it remove the operator. `GET_DISTINCT_IDS_BY_PROPERTY_SQL` doesn't need an operator because i removed the dummy `1 = 1`